### PR TITLE
Add ESLint, Prettier and linting git hook

### PR DESCRIPTION
**Pull request by NatanCieplinski**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,12 +33,19 @@
...
+    "deploy": "vercel deploy --local-config=vercel.json --prod",
+    "lint": "eslint --ext js,ts,tsx src",
+    "lint:fix": "npm run lint -- --fix"
   },
   "eslintConfig": {
     "extends": [
       "react-app",
-      "react-app/jest"
+      "react-app/jest",
+      "prettier",
```
Comment (mmalfertheiner) [inline package.json:44]: I think this line can be removed, according to the docs: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration

**Review by mmalfertheiner — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (NatanCieplinski) [reply package.json:44]: In theory yes, in practice I did it and now i'm fighting with Prettier, that is not recognizing some allowed keywords

Comment (mmalfertheiner) [reply package.json:44]: Where does it happen for you? I tried it locally and it works just fine.

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+yarn.json
+.next
+node_modules/
+out
```
Comment (mmalfertheiner) [inline .prettierignore:4]: Here I would only put:

```
build
node_modules
```
- We are not using yarn.
- out and .next are folders created by "Next.js"

**Review by mmalfertheiner — APPROVED** [review]

**Pull request merged**